requirements.txt

```diff
@@ -1,4 +1,4 @@
-pytest==3.0.1
+pytest==3.0.2
```

Comment:
Did you want to consider trying a compatible release version specifier such as `pytest~=3.0.0` which would pick up bugfix releases but not changes in functionality?